treeherder/utils/taskcluster.py

```diff
@@ -1,8 +1,28 @@
 import taskcluster_urls
+import yaml
 
-from treeherder.utils.http import fetch_json
+from treeherder.utils.http import fetch_json, fetch_text, make_request
 
 
 def get_task_definition(root_url, task_id):
     task_url = taskcluster_urls.api(root_url, 'queue', 'v1', 'task/{}'.format(task_id))
     return fetch_json(task_url)
+
+
+def download_artifact(root_url, task_id, path):
```

Comment:
You may be able to test this method by using `responses`, and check that it returns a valid Python structure when loading a `.json` file